### ISSUE TITLE
convolutional auto encoder train and test functions and scripts 

### ISSUE THREAD

**Issue by tyiannak** [tyiannak/deep_audio_features#23]

**Comment by rodo-K** [comment]
Regarding "convolutional_autoencoder" branch, I added 3 files:
bin/basic_ae_train.py                                                                                                                                                                                   lib/ae_training.py                                                                                                                                                                                     models/ae_cnn.py

When I'm trying to run bin/basic_ae_train.py ( as the bin/basic_training.py runs) I'm getting wrong dimensions from the decoder part as you can see below:

_Using a target size (torch.Size([16, 1, 128, 201])) that is different to the input size (torch.Size([16, 1, 129, 201])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size._

Please let me know what I should check and If the initial try of the autoencoder is ok besides this error.

**Comment by rodo-K** [comment]
after calculating the right padding, stride etc. Got output 201 x 128. Somehow still not getting the result I want from the loss function. Can't figure out why. 

![Screenshot 2021-07-25 154509](https://user-images.githubusercontent.com/78028914/126899432-f65930ce-ea7b-43a9-bb9b-4452b2415747.png)
